tests/main/preseed-reset/task.yaml

```diff
@@ -3,7 +3,7 @@ description: |
   This test checks that preseeding of Ubuntu cloud images with snap-preseed
   can be undone with --reset flag.
 
-systems: [ubuntu-19.10-*, ubuntu-2*]
+systems: [ubuntu-19.10-*, ubuntu-20*]
```

Comment:
I think we want a TODO here as well, I guess the same/similar to the one for lxd?

Reply:
My understanding (after calrifying qemu-utils confusion on 20.10) was that these tests pass on 20.10? If not, then I agree, please add a comment explaining why 20.10 is excluded.